src/data/propertyPaths.ts

```diff
@@ -21,7 +37,7 @@ export const parsedPropertyPaths: Record<string, string[]> = {
     'subtitle',
     'status',
     'experimental',
-    'subject',
+    'subject[x]',
```

Comment:
OOooO choice types!! Hmm. Let's chat about this tomorrow. Not sure if this is something we should account for in this script (probably) but maybe not. I think it's probably easiest to deal with in this script though and won't be too hard.